src/vaadin-dialog-draggable-mixin.html

```diff
@@ -0,0 +1,47 @@
+<script>
+  /**
+   * @namespace Vaadin
+   */
+  window.Vaadin = window.Vaadin || {};
+
+  /**
+   * @polymerMixin
+   * @memberof Vaadin
+   */
+  Vaadin.DialogDraggableMixin = subclass => class VaadinDialogDraggableMixin extends subclass {
+    ready() {
+      super.ready();
+      this._addDraggableEventListeners();
+    }
+
+    _addDraggableEventListeners() {
+      let originalBounds;
+      let originalMouse;
+      const stopDrag = () => {
+        window.removeEventListener('mouseup', stopDrag);
+        window.removeEventListener('mousemove', drag);
+      };
+      const drag = (e) => {
+        this._snapToPixels();
```

Comment:
Shouldn't this be only done once in dragStart?

Reply:
Yes true, I changed it now.